common_features.mk

```diff
@@ -380,6 +380,12 @@ ifeq ($(strip $(VIA_ENABLE)), yes)
     OPT_DEFS += -DVIA_ENABLE
 endif
 
+ifeq ($(strip $(ORGB_ENABLE)), yes)
```

Comment:
Let's name this `OPENRGB_ENABLE` for clarity.

Reply:
Should probably do that in most places, too. `orgb.c` => `openrgb.c`, typedefs too.

Reply:
Done